README.md

```diff
@@ -17,12 +17,10 @@ Desktop Frontend of Qabel
    ```
 0. [start the servers](https://github.com/Qabel/qabel/blob/master/README.md#starting-the-servers) in a new terminal
 
-0. run (example with helloworld-module)
-
-   ```
-   java -Djava.library.path=../qabel-core -cp "../qabel-helloworld-module/build/libs/qabel-helloworld-module-0.1.jar:../qabel-desktop/build/libs/qabel-desktop-0.1.jar" de.qabel.desktop.QblMain -module qabel-helloworld-module/build/libs/qabel-helloworld-module-0.1.jar:de.qabel.helloworld.QblHelloWorldModule
-   ```
-
 ### all-in-one build
 
 travis uses the build.sh. it starts the required servers, runs the tests and stops the servers afterwards.
+
+## running Qabel Desktop
+
+A simple `./gradlew run` does not work. Use `./gradlew distZip` instead, unzip it and copy the curve library file (e.g. `qabel-core/build/binaries/curve25519SharedLibrary/osx_amd64/libcurve25519.dylib`) into the libs folder of the unzipped dist file prior to running the application.
```

Comment:
And I think you are right but you misunderstood the intention of a pull request. A pull request can be a solution to a given problem, a base for a discussion about a topic even never mentioned before or a suggestion for something.

This PR was a quick fix for the problem that the `README.md`file is *so* wrong that it misleads a developer trying to actually help. Of course this PR is not a good solution to anything. It merely shows a problem while providing a quick fix.

I agree to your approach. Nevertheless you should write your `README.md` in a way it actually helps. And to be clear about this, the problem with the documentation has nothing to do with the platform (darwin, linux etc.).

The problem is even greater since the readme files from the other repositories are misleading, too.